workers/unity/Packages/com.improbable.gdk.deploymentlauncher/DeploymentLauncherWindow.cs

```diff
@@ -168,32 +224,39 @@ private void OnGUI()
                     }
                 }
 
-                using (new GUILayout.HorizontalScope())
+                if (launcherConfig.DeploymentConfigs.Count > 0)
                 {
-                    selectedDeploymentIndex = EditorGUILayout.Popup("Deployment", selectedDeploymentIndex,
-                        launcherConfig.DeploymentConfigs.Select(config => config.Deployment.Name).ToArray());
+                    using (new GUILayout.HorizontalScope())
+                    {
+                        selectedDeploymentIndex = EditorGUILayout.Popup("Deployment", selectedDeploymentIndex,
+                            launcherConfig.DeploymentConfigs.Select(config => config.Deployment.Name).ToArray());
 
-                    var isValid = selectedDeploymentIndex >= 0 &&
-                        selectedDeploymentIndex < launcherConfig.DeploymentConfigs.Count;
+                        var isValid = selectedDeploymentIndex >= 0 &&
```

Comment:
you use this also a couple of time. maybe factor it out and just call something like `SelectedValidDeployment()`

Reply:
`IsSelectedValid(List<T> items, int index)` ?